src/slo.ts

```diff
@@ -29,7 +29,8 @@ function getRepoResults(repos: IssueResult[]) {
       p2: 0,
       pX: 0,
       outOfSLO: 0,
-      repo: repo.repo.repo
+      repo: repo.repo.repo,
+      repoLanguage: repo.repo.language
```

Comment:
`language` instead of `repoLanguage` is probably fine here

Reply:
Done